src/takeLastWhile.ts

```diff
@@ -0,0 +1,46 @@
+import { purry } from './purry';
+
+/**
+ * Returns elements from the end of the array until the predicate returns false.
+ *
+ * @param array the array
+ * @param fn the predicate
+ * @signature
+ *    R.takeLastWhile(array, fn)
+ * @example
+ *    R.takeLastWhile([1, 2, 3, 4, 3, 2, 1], x => x !== 4) // => [3, 2, 1]
+ * @dataFirst
+ * @category Array
+ */
+export function takeLastWhile<T>(
+  array: ReadonlyArray<T>,
+  fn: (item: T) => boolean
+): Array<T>;
```

Comment:
Prefer descriptive names
```suggestion
export function takeLastWhile<T>(
  data: ReadonlyArray<T>,
  predicate: (item: T) => boolean
): Array<T>;
```

Reply:
Ok :+1: My idea was to keep the naming and docs consistent with `takeWhile` (and most other functions).

Done.